bolt/lib/Profile/BoltAddressTranslation.cpp

```diff
@@ -160,12 +164,31 @@ std::error_code BoltAddressTranslation::parse(StringRef Buf) {
     return make_error_code(llvm::errc::io_error);
 
   Error Err(Error::success());
+  std::vector<uint64_t> HotFuncs;
```

Comment:
Is this HotFuncs vector going to be used in another diff of this stack? It looks unused to me atm.

Reply:
It's populated by `parseMaps<Hot>` and read by `parseMaps<Cold>`.
Cold functions encode indices of parent hot functions (also delta encoded), hence this vector is required.